build/xilinx/symbiflow.py:  dependence error seen with parallel make

The SymbiFlow build uses "make".   The generated makefile contains these lines:

```
$(TOP).eblif: $(VERILOG) $(MEM_INIT) $(XDC)
        symbiflow_synth -t $(TOP) -v $(VERILOG) -d $(BITSTREAM_DEVICE) -p $(PARTNAME) -x $(XDC) > /dev/null

$(TOP).net: $(TOP).eblif $(SDC)
        symbiflow_pack -e $(TOP).eblif -d $(DEVICE) -s $(SDC) > /dev/null
```

When the "make" is allowed to have multiple parallel jobs, and is trying to build `$(TOP).net`, it will try to build both dependencies `$(TOP).eblif` and `$(SDC)` in parallel.   But there is no rule to build `$(SDC)`, and it doesn't exist yet, because it is built as a side effect of building `$(TOP).eblif`.  This causes an error.

To reproduce, get SymbiFlow in your environment, and then in litex_boards/targets, run:
```
MAKEFLAGS=-j8 ./digilent_arty.py --toolchain=symbiflow --build --load
```
The actual error will be:
```
ROM usage: 26.66KiB     (20.83%)
RAM usage: 1.70KiB      (21.19%)

make: Leaving directory '/home/tim/enjoy-digital/litex-boards/litex_boards/targets/build/digilent_arty/software/bios'
INFO:SoC:Initializing ROM rom with contents (Size: 0x6aa8).
INFO:SoC:Auto-Resizing ROM rom from 0x20000 to 0x6aa8.
make: *** No rule to make target 'digilent_arty.sdc', needed by 'digilent_arty.net'.  Stop.
make: *** Waiting for unfinished jobs....
```

Possible fixes:

* Explicitly invoke "make" with "-j1" to disable parallel jobs for this make.
* Remove the `$(SDC)` dependence.

--- a/litex/build/xilinx/symbiflow.py
+++ b/litex/build/xilinx/symbiflow.py
@@ -73,7 +73,7 @@ class _MakefileGenerator:
 
 
 def _run_make():
-    make_cmd = ["make"]
+    make_cmd = ["make", "-j1"]
 
     if which("symbiflow_synth") is None:
         msg = "Unable to find Symbiflow toolchain, please:\n"

